packages/react/src/components/Table/TableHead/FilterHeaderRow/FilterHeaderRow.jsx

```diff
@@ -60,6 +60,7 @@ class FilterHeaderRow extends Component {
         ),
         /** if isMultiselect and isFilterable are true, the table is filtered based on a multiselect */
         isMultiselect: PropTypes.bool,
+        customInput: PropTypes.elementType,
```

Comment:
add a fully description of this prop so that when we read the docs we know how to use it, and what to expect for input and outputs, etc.